.env.local

```diff
@@ -1,3 +1,4 @@
 BROWSER=none
 EXTEND_ESLINT=true
-FAST_REFRESH=true
\ No newline at end of file
+FAST_REFRESH=true
+PORT=4000
```

Comment:
Preferably, the environment variable for the port should be in `.env.development.local`.